policy/modules/services/cron.te

```diff
@@ -422,6 +422,7 @@ optional_policy(`
 
 optional_policy(`
 	systemd_dbus_chat_logind(crond_t)
+	systemd_dontaudit_connect_machined(crond_t)
```

Comment:
I'm curious, why is it trying to connect?

Reply:
I don't know either. I'll remove it now and maybe submit in a different PR